content/posts/2023/reana-0.9.0.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "REANA 0.9.0 is released"
+date: 2023-01-24T09:00:00+01:00
+---
+
+We are happy to announce REANA 0.9.0, a major update that allows
+users to set custom workflow retention rules to automatically delete workspace
+files, adds a new way of launching workflows from external sources, and more!
+
+<!--more-->
+
+## What's new for the users?
+
+### Automatically delete workspace files using custom retention rules
+
+It's helpful to have a way to automatically delete files in your workflow's
+workspace that you don't need anymore. For example, if your workflow creates
+large temporary files that take up a lot of space that consume your
+[disk quota](https://docs.reana.io/advanced-usage/user-quotas), you may want to
+get rid of them after a certain period of time. Therefore, we're introducing the
+ability to set up custom rules for keeping or deleting files in your workspace.
+This can save you from having to manually delete them, which can be
+time-consuming process.
+
+![Retention rules in the web UI](/images/ui-retention-rules.png)
+
+Please see a dedicated blog post about [custom file retention rules](https://blog.reana.io/posts/2022/workspace-file-retention-rules/)
+to learn more about this feature.
+
+### Launching workflows from external sources using web interface
+
+You can now easily run your analysis workflows from publicly accessible external
+sources, like GitHub, GitLab, digital repositories like Zenodo or any website on
+the web using the new REANA launcher page. You can host your workflows on a
+public location and be able to access and run the workflow on the REANA web site
+without having to do any manual steps.
+
+[![Launch home](/images/reana-0.9.0-launcher.png)](https://reana.cern.ch/launch)
+
+For more information about this new feature please check a dedicated blog post
+about [launching workflows from external sources using web interface](https://blog.reana.io/posts/2022/launching-workflows).
+
+### Rucio authentication for workflow jobs
+
+[Rucio](https://rucio.cern.ch/) is a scientific data management system used in
+LHC particle physics and related scientific domains. It provides access for
+large volumes of data spread across facilities at multiple institutions.
+
+If your workflow needs to use data that is managed by a Rucio system, you can
+follow the documentation that describes the [Rucio authentication method](https://docs.reana.io/advanced-usage/access-control/rucio/)
+to learn more about it.
+
+### Kerberos authentication for workflow orchestration
+
+Previously, you had to set up Kerberos authentication for each step of your
+workflow one at a time. Finally you can easily enable Kerberos authentication
+for your entire workflow by adding just one line in the specification file. This
+makes it much more convenient and efficient.
```

Comment:
I think we should briefly explain that this is also useful for accessing restricted resources during the workflow orchestration. What do you think? One sentence would be enough, and we can also link to the related blog post (not published yet, though).

Reply:
I've added one sentence to explain it, but haven't changed the link to the blog post yet, since it's not clear if we will merge it today or not. I will update it later once it's clear